packages/validator/src/plugins/validation/2and3/semantic-validators/schema-ibm.js

```diff
@@ -155,56 +155,19 @@ function generateFormatErrors(schema, contextPath, config, isOAS3, messages) {
     return;
   }
 
-  // Special case: check for arrays of arrays
```

Comment:
My goal in this file was to remove the logic associated ONLY with the two new rules (array-of-arrays, valid-type-format sans file type).